api/user.json

```diff
@@ -0,0 +1,3 @@
+{
+  name: "Penelope Morar"
```

Comment:
Oops..  Right you are.  It was just copied and pasted from the back-end from a console.log, hence the invalid json.  Fixing up now.